target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -141,7 +141,9 @@ case "$FIRMWARE" in
 		ath10kcal_extract "ART" 4096 12064
 		ath10kcal_patch_mac_crc $(mtd_get_mac_binary dnidata 0)
 		;;
+	engenius,eap1300 |\
 	compex,wpj428 |\
+	engenius,eap1300 |\
```

Comment:
This seems to be double-defined?

Reply:
Yes, It looks that way. There are also two patches with the same name
"ipq40xx: add support for EnGenius EAP1300" in the PR as well as a
glue patch "Merge branch 'master' into add-support-for-EnGenius-EAP1300".
I think the OpenWrt project owners are waiting for @gamism to fix the two engenius,eap1300 entries and squash the three patches together into a single "ipq40xx: add support for EnGenius EAP1300" patch.